src/agent/LibFuzzerDotnetLoader/Program.cs

```diff
@@ -81,4 +75,63 @@ static bool TryTestOne<T>(MethodInfo method, Func<T, SharpFuzz.ReadOnlySpanActio
             return TryTestOne<TestOneArray>(method, t => span => t(span.ToArray()));
         }
     }
+
+    class LibFuzzerDotnetTarget
+    {
+        public string AssemblyPath { get; private set; }
+        public string ClassName { get; private set; }
+        public string MethodName { get; private set; }
```

Comment:
private set is not needed since you only assign those in the constructor
```suggestion
        public string AssemblyPath { get;}
        public string ClassName { get;}
        public string MethodName { get;}
```

Reply:
Ah, right, is really only necessary for non-`public` properties if we want to make it `protected`.